pokete.py

```diff
@@ -681,11 +765,18 @@ def give_item(self, item, amount=1):
         logging.info("[Figure] %d %s(s) given", amount, item)
 
     def has_item(self, item):
-        """Checks if an item is already present"""
+        """Checks if an item is already present
+        ARGS:
+            item: Generic item name
+        RETURNS:
+            bool"""
```

Comment:
All other methods say what they return and not the type:

```suggestion
            If the player has this item"""
```